test/integration_test.js

```diff
@@ -766,26 +846,35 @@ describe("Integration tests", function() {
       describe("Outgoing conflict", () => {
         let syncResult;
 
-        beforeEach(() => {
+        function setupConflict(collection) {
           let recordId;
-          return fetch(`${TEST_KINTO_SERVER}/buckets/default/collections/tasks/records`, {
-            method: "POST",
-            headers: {
-              "Accept":        "application/json",
-              "Content-Type":  "application/json",
-              "Authorization": "Basic " + btoa("user:pass"),
-            },
-            body: JSON.stringify({data: {title: "task1-remote", done: true}})
+          const record = {title: "task1-remote", done: true};
+          // Ensure that the remote record looks like something that's
+          // been transformed
+          return collection._encodeRecord("remote", record).then(record => {
+            return fetch(`${TEST_KINTO_SERVER}/buckets/default/collections/${collection._name}/records`, {
```

Comment:
Ah, that's a good point. I wasn't looking at it too closely, I thought it *was* the kinto-http API.